src/SimpleGraphs/simpledigraph.jl

```diff
@@ -26,6 +26,31 @@ end
 
 eltype(x::SimpleDiGraph{T}) where {T} = T
 
+# SimpleDiGraph(g)
+"""
+    SimpleDiGraph{T}(g::AbstractGraph)
+
+Construct a `SimpleDiGraph{T}` from an `AbstractGraph`. 
+If `g` is undirected, then every undirected edge x - y in `g` is added 
+as two directed edges x -> y and y -> x.
+"""
+SimpleDiGraph(g::AbstractGraph) = SimpleDiGraph{Int}(g)
```

Comment:
```suggestion
SimpleDiGraph(g::AbstractGraph) = SimpleDiGraph{eltype(g)}(g)
```
or
```suggestion
SimpleDiGraph(g::AbstractGraph{T}) where {T} = SimpleDiGraph{T}(g)
```
see below for the reason